CONTRIBUTING.md

```diff
@@ -32,14 +45,16 @@ How to get your contributions merged smoothly and quickly.
 - Provide a good **PR description** as a record of **what** change is being made
   and **why** it was made. Link to a github issue if it exists.
 
-- Don't fix code style and formatting unless you are already changing that line
-  to address an issue. PRs with irrelevant changes won't be merged. If you do
-  want to fix formatting or style, do that in a separate PR.
+- Don't fix code style, typos, and formatting unless you are already changing
+  that line to address an issue. PRs with irrelevant changes won't be merged. If
+  you do want to fix formatting or style, do that in a separate PR.
```

Comment:
I'd like to see this updated further.

> If you want to fix formatting or style, consider whether your changes are an obvious improvement or might be considered a personal preference.  If a style change is based on preference, it likely will not be accepted.  If it corrects widely agreed-upon anti-patterns, then please do create a PR and explain the benefits of the change.

Or something like that.

Reply:
Done.